Documentation/building/windows-instructions.md

```diff
@@ -8,44 +8,81 @@ MSBuild version 15 or later is required.
 
 # Building Xamarin.Android
 
- 1. Install the [build dependencies](dependencies.md).
+ 1. Run the [Visual Studio Installer](https://visualstudio.microsoft.com/vs/).
 
- 2. Clone the xamarin-android repo:
+ 2. Under the **Workloads** tab, ensure that the **Mobile development with
+    .NET** workload is installed.  Under the **Optional** items for the
+    workload, ensure **Android SDK setup** is selected.
+
+ 3. Also ensure the **.NET desktop development** workload is installed.  Under
+    the **Optional** items for the workload, ensure the following items are
+    installed:
+
+      * **.NET Framework 4 &ndash; 4.6 development tools**
+      * **.NET Framework 4.6.2 development tools**
+      * **.NET Framework 4.7 development tools**
+      * **.NET Core 2.0 development tools**
+
+    The following items are also recommended:
+
+      * **.NET Framework 4.7.1 development tools**
+      * **.NET Framework 4.7.2 development tools**
```

Comment:
Ugh, so mental note we should standardize all `$(TargetFrameworkVersion)` if we can. So we don't have to install all of these...

We need to get on 4.7.2 anyway, because of changes in MSBuild in VS 2019.